hypnos/src/event_handler.c

```diff
@@ -109,13 +118,17 @@ void battery_charging_isr(struct device *gpiobat, struct gpio_callback *cb, u32_
         gpio_pin_read(charging_dev, 12, &res);
         battery_update_charging_status(res != 1U);
 	backlight_enable(true);
-	k_timer_start(&backlight_off_timer, BACKLIGHT_OFF_TIME, 0);
+	k_timer_start(&backlight_off_timer, BACKLIGHT_TIMEOUT, 0);
 }
 
 void button_pressed_isr(struct device *gpiobtn, struct gpio_callback *cb, u32_t pins)
 {
 	backlight_enable(true);
-	k_timer_start(&backlight_off_timer, BACKLIGHT_OFF_TIME, 0);
+	k_timer_start(&backlight_off_timer, BACKLIGHT_TIMEOUT * 4, 0);
```

Comment:
Why multiply by 4? Make a new alias instead, to make it easier to adjust.

Reply:
I removed the multiplication; it was just there for debugging. So, fixed